server/chat-commands/core.ts

```diff
@@ -1532,12 +1533,12 @@ export const commands: ChatCommands = {
 			}
 			interface RoomData {p1?: string; p2?: string; isPrivate?: boolean | 'hidden' | 'voice'}
 			let roomList: {[roomid: string]: RoomData} | false = {};
-			for (const roomid of targetUser.inRooms) {
-				const targetRoom = Rooms.get(roomid);
-				if (!targetRoom) continue; // shouldn't happen
+			const targetRooms = [...Rooms.rooms.values()].filter(cur => targetUser.id in cur.users);
+			for (const targetRoom of targetRooms) {
```

Comment:
```suggestion
			for (const targetRoom of targetUser.getRooms()) {
```
Am I missing something? Is there a reason you didn't do this?